src/index.js

```diff
@@ -2,18 +2,17 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
+import 'bootstrap/dist/css/bootstrap.min.css';
 import reportWebVitals from './reportWebVitals';
 
-import { BrowserRouter } from 'react-router-dom'
-
 const routs = (
```

Comment:
if you want you can correct the spelling of routes variable, else it's fine.

Reply:
ok